@kiva/kv-components/vue/KvCarousel.vue

```diff
@@ -0,0 +1,255 @@
+					tw-h-4 tw-w-4
+					tw-flex tw-items-center tw-justify-center"
+				:disabled="embla && !embla.canScrollPrev()"
+				@click="handleUserInteraction(previousIndex, 'click-left-arrow')"
+			>
+				<kv-material-icon
+					class="tw-w-4"
+					:icon="mdiChevronLeft"
+				/>
+				<span class="tw-sr-only">Show previous slide</span>
+			</button>
+			<div class="tw-mx-2 md:tw-mx-3 lg:tw-mx-4 tw-invisible md:tw-visible">
+				{{ currentIndex + 1 }}/{{ slideIndicatorList.length }}
+			</div>
+			<button
+				class="tw-text-gray-800 disabled:tw-text-gray-300
+					tw-rounded-full
+					tw-border-2 tw-border-text-gray-800
+					tw-h-4 tw-w-4
+					tw-flex tw-items-center tw-justify-center"
+				:disabled="embla && !embla.canScrollNext()"
+				@click="handleUserInteraction(nextIndex, 'click-right-arrow')"
+			>
+				<kv-material-icon
+					class="tw-w-4"
+					:icon="mdiChevronRight"
+				/>
+				<span class="tw-sr-only">Show next slide</span>
+			</button>
+		</div>
+	</section>
+</template>
+
+<script>
+import EmblaCarousel from 'embla-carousel';
+import { mdiChevronLeft, mdiChevronRight } from '@mdi/js';
+import { throttle } from '../utils/throttle';
+
+import KvMaterialIcon from './KvMaterialIcon.vue';
+import KvCarouselSlide from './KvCarouselSlide.vue';
+
+export default {
+	components: {
+		KvMaterialIcon,
+		KvCarouselSlide,
+	},
+	props: {
+		/**
+		 * Should multiple slides be visible at a time.
+		 * Used to set margin left/right for an individual carousel slide.
+		 * */
+		multipleSlidesVisible: {
+			type: Boolean,
+			default: false,
+		},
+		/**
+		 * Number of slides the carousel contains
+		 * */
+		numberOfSlides: {
+			type: Number,
+			default: 0,
+			required: true,
+		},
+		/**
+		 * Options for the embla carousel - // https://davidcetinkaya.github.io/embla-carousel/api#options
+		 * */
+		emblaOptions: {
+			type: Object,
+			default() {
+				return {};
+			},
+		},
+		/**
+		 * The type of logic to implement when deciding how many slides
+		 * to scroll when pressing the next/prev button
+		 * `visible, auto`
+		 * */
+		slidesToScroll: {
+			type: String,
+			default: 'auto',
+			validator: (value) => ['visible', 'auto'].indexOf(value) !== -1,
+		},
+	},
+	data() {
+		return {
+			mdiChevronLeft,
+			mdiChevronRight,
+			embla: null,
+			slides: [],
+			currentIndex: 0,
+		};
+	},
+	computed: {
+		nextIndex() {
+			const nextSlideIndex = this.currentIndex + 1;
+			if (nextSlideIndex < this.slides.length) {
+				return nextSlideIndex;
+			}
+			return 0;
+		},
+		previousIndex() {
+			const previousSlideIndex = this.currentIndex - 1;
+			if (previousSlideIndex >= 0) {
+				return previousSlideIndex;
+			}
+			return this.slides.length - 1;
+		},
+		slideIndicatorList() {
+			return this.embla ? this.embla.scrollSnapList() : 0;
+		},
+	},
+	mounted() {
+		// initialize Embla
+		this.embla = EmblaCarousel(this.$refs.KvCarousel, {
+			loop: true,
+			containScroll: 'trimSnaps',
+			inViewThreshold: 0.1,
```

Comment:
I had to change this `inViewThreshold` to be able to determine which slides are visible to set aria-hidden property.

Reply:
I had to change the ViewThreshold the low threshold wont work with the multiple slides functionality. This means that a slide partially in view gets the aria-hidden attribute added to it. I think this is fine for a11y